BodoSQL/calcite_sql/bodosql-calcite-application/src/main/java/org/apache/calcite/sql/validate/SqlValidator.java

```diff
@@ -923,6 +925,27 @@ public interface Config {
      */
     Config withIdentifierExpansion(boolean expand);
 
+    /**
+     * Returns whether to treat the query being validated as embedded
+     * (as opposed to top-level).
+     *
+     * <p>The default, false, treats the query as top-level;
+     * a value of true treats it as a query inside another, as would be the case
+     * for a view or common table expression.
+     *
+     * <p>Possible behavior differences include ignoring the {@code ORDER BY}
+     * clause of an embedded query, or converting measure expressions of a
+     * non-embedded query into values. */
+    @Value.Default default boolean embeddedQuery() {
```

Comment:
Let's file an issue to explore this. We might have a simpler/better view expansion process now.